db/migrate/20170419121747_create_restaurants.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRestaurants < ActiveRecord::Migration[5.0]
+  def change
+    create_table :restaurants do |t|
+      t.string :name
+      t.string :genre
+      t.integer :rating
+      t.boolean :tenbis
+      t.string :address
```

Comment:
What's the structure of an address? Is it just a string, or something more complex (city, street, number)?